diff --git a/node_modules/@eui/ecl/fesm2022/eui-ecl.mjs b/node_modules/@eui/ecl/fesm2022/eui-ecl.mjs
index d63a29b..d64a3e7 100644
--- a/node_modules/@eui/ecl/fesm2022/eui-ecl.mjs
+++ b/node_modules/@eui/ecl/fesm2022/eui-ecl.mjs
@@ -44,7 +44,7 @@ import '@angular/common/locales/global/sl';
 import '@angular/common/locales/global/sv';
 import Pikaday from 'pikaday';
 import * as i1$5 from '@angular/platform-browser';
-import Hammer from 'hammerjs';
+import Hammer from '@egjs/hammerjs';
 import { ENTER } from '@angular/cdk/keycodes';
 
 class EclBaseEvent {
