diff --git a/node_modules/pikaday/pikaday.js b/node_modules/pikaday/pikaday.js
index 6164245..7bce116 100755
--- a/node_modules/pikaday/pikaday.js
+++ b/node_modules/pikaday/pikaday.js
@@ -33,15 +33,14 @@
     /**
      * feature detection and helper functions
      */
-    var hasMoment = typeof moment === 'function',
+    var hasMoment = typeof moment === 'function';
+    if (typeof window !== 'undefined') {
+        var hasEventListeners = !!window.addEventListener
 
-    hasEventListeners = !!window.addEventListener,
-
-    document = window.document,
-
-    sto = window.setTimeout,
+        var sto = window.setTimeout;
+    };
 
-    addEvent = function(el, e, callback, capture)
+    var addEvent = function(el, e, callback, capture)
     {
         if (hasEventListeners) {
             el.addEventListener(e, callback, !!capture);
